Python.gitignore

```diff
@@ -1,4 +1,9 @@
-# Byte-compiled / optimized / DLL files
+ .idea/
+ .pytest_cache
+ *.__pycache__/
+ env/
```

Comment:
Is this created by default or has some special meaning for Python? If it's just a convention or your preference to create this folder, it shouldn't be included here